gliner/model.py

```diff
@@ -831,7 +834,12 @@ def _from_pretrained(
                 session_options.graph_optimization_level = (
                     ort.GraphOptimizationLevel.ORT_ENABLE_ALL
                 )
-            ort_session = ort.InferenceSession(model_file, session_options)
+            providers = ['CPUExecutionProvider']
+            if map_location == 'cuda':
```

Comment:
map_location could be "cuda:0" ?

Reply:
Yeah, it will be more general to make it like that:
`if "cuda" in map_location`

Reply:
great catch, let me fix that

Reply:
done